### PR TITLE
Pbi/38285 add faq to reporting installation documentation

### PR DESCRIPTION
**Pull request by KabbaniReem**
Here are updates concerning installation of reporting locally by adding more info and updating FAQ with some problems and corresponding solutions for.

### CODE REVIEW

**Review by JochenLinnemann — COMMENTED** [review]
I don't like mixing topics in a pull request. Here we have permission and reporting in one PR...

**Comment by KabbaniReem** [comment]
> I don't like mixing topics in a pull request. Here we have permission and reporting in one PR...
This was not ment to be this way, I thought I have made the PR to be merged with master. Sorry for that!

**Comment by KabbaniReem** [comment]
I reverted the changes in commits about reporting since they are already in master, should this cause any problems? @JochenLinnemann

**Review by Gerrit-P-und-Z — DISMISSED** [review]

File: docs/admin/services/reporting-service/faq.md [hunk 1]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
 #### Symbio Core
 
 - Can Symbio Core access the Reporting Connector? Try to open the configured *Service Endpoint URL* of the Report Pool (External System in Symbio SysAdmin) in a browser on the server machine hosting Symbio Core.
-  - Can the account Symbio Core is running under do the same?
+  - Can the account that is running under Symbio Core do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:79]: Can the account that is running Symbio Core do the same?

File: docs/admin/services/reporting-service/faq.md [hunk 2]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
   - Does the time update on refresh?
   - Is the version number correct?
 
 - Can Symbio Core access the targeted SSRS instance (see Reporting Connector configuration)? Try to open the ReportPortal URL in a browser on the server machine hosting Symbio Core.
-  - Can the account Symbio Core is running under do the same?
+  - Can the account that is running under Symbio Core do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:84]: strike "under"

File: docs/admin/services/reporting-service/faq.md [hunk 3]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
 
 - Can the Reporting Connector access Symbio Core? Try to open the Symbio Core URL in a browser on the server machine hosting the Reporting Connector.
-  - Can the account the Reporting Connector is running under do the same?
+  - Can the account that is running under the Reporting Connector do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:87]: strike "under"

File: docs/admin/services/reporting-service/faq.md [hunk 4]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
 
 #### Reporting Connector
 
 - Can the Reporting Connector access the targeted SSRS instance (see Reporting Connector configuration)? Try to open the ReportServer URL in a browser on the server machine hosting the Reporting Connector.
-  - Can the account the Reporting Connector is running under do the same?
+  - Can the account that is running under the Reporting Connector do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:92]: strike "under"

File: docs/admin/services/reporting-service/faq.md [hunk 5]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
 
-- Can the Reporting Connector access the ODS database? Try to connect to the ODS database using a command line tool or SQL Server Management Studio on the server machine hosting the Reporting Connector.
-  - Can the account the Reporting Connector is running under do the same?
+- Can the Reporting Connector access the ODS database? Try to connect to the ODS database using a command line tool or SQL Server Management Studio on the server machine hosting the Reporting Connector. (One requirement that you should be aware of is that under Security in the Server Properties, SQL Sever and Windows Authentication mode should be selected)
+  - Can the account that is running under the Reporting Connector do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:95]: strike "under"

File: docs/admin/services/reporting-service/faq.md [hunk 6]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
 
 #### Reporting Console
 
 - Can the Reporting Console access Symbio Core? Try to open the Symbio Core URL in a browser on the machine running scheduled Reporting Console tasks.
-  - Can the account scheduled tasks are running under do the same?
+  - Can the account that is running under scheduled tasks do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:100]: strike "under"

File: docs/admin/services/reporting-service/faq.md [hunk 7]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
 
 - Can the Reporting Console access the Reporting Connector? Try to open the configured *Service Endpoint URL* of the Report Pool (External System in Symbio SysAdmin) in a browser on the machine running scheduled Reporting Console tasks.
-  - Can the account scheduled tasks are running under do the same?
+  - Can the account that is running under scheduled tasks do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:103]: strike "under"

File: docs/admin/services/reporting-service/faq.md [hunk 8]
```diff
@@ -72,36 +76,36 @@ If error messages indicate a connection problem, try the following (depending on
...
   - Does the time update on refresh?
   - Is the version number correct?
 
 - Can the Reporting Console access the ODS database? Try to connect to the ODS database using a command line tool or SQL Server Management Studio on the machine running scheduled Reporting Console tasks.
-  - Can the account scheduled tasks are running under do the same?
+  - Can the account that is running under scheduled tasks do the same?
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/faq.md:108]: strike "under"

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 9]
```diff
@@ -3,19 +3,19 @@
...
 
-To set up the **Reporting Connector** you will need either an IIS installation with a current version of the ASP.NET Core Hosting package installed or an Azure App Service ready for deployment. In both cases the service will need access to Symbio, SSRS and the ODS database. It must also be reachable by the server side component of Symbio.
+To set up the **Reporting Connector** you will need either an IIS installation with a current version of the **[ASP.NET Core Runtime](https://dotnet.microsoft.com/download)** package installed or an Azure App Service ready for deployment. In both cases the service will need access to Symbio, SSRS and the ODS database. It must also be reachable by the server side component of Symbio.
 
 ## Installation
 
 #### Reporting Connector on IIS
 1. Create a directory to host the files of the connector service, e.g. _C:\SymbioServices\Reporting\Connector_
-2. Copy all files and folders of the service into this directory
+2. After building the reporting project, which contains Conector Service, Copy all the debug/release files and folders of the service into this directory.
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-connector.md:12]: The customer will NOT build. This documentation is customer-facing...

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 10]
```diff
@@ -3,19 +3,19 @@
...
 3. In IIS Manager:
-    1. Create a new Application pool (No Managed Code/Integrated) and assign a user account with appropriate rights to the created directory
-    2. Create a new Application using the created pool and selecting the created directory
-    3. Note the resulting URL (composed of website binding URL and the selected alias/path of the new application)
-4. Configure the Connector by editing its appsettings.config files 
-5. Test connection to service by navigating to the noted URL
+    1. Create a new Application pool (No Managed Code/Integrated) and assign a user account with appropriate rights (full Control) to the created directory in step 1.
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-connector.md:14]: "Full Control" or "full control"

File: docs/admin/services/reporting-service/reporting-connector.md [hunk 11]
```diff
@@ -3,19 +3,19 @@
...
+    2. Create a new Application (Web Site) using the created pool and selecting the created directory.
+    3. Note the resulting URL (composed of website binding URL and the selected alias/path of the new application).
+4. Configure the Connector by editing its appsettings.config files (can be found among the resulting build files).
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-connector.md:17]: The customer has no "build" files

File: docs/admin/services/reporting-service/ssrs-and-reports.md [hunk 12]
```diff
@@ -17,7 +17,8 @@ Please secure your Report Service with an HTTPS endpoint, choose a valid and acc
 
 ### Database
 
-The Report server stores all report server content and application data in a database. This is the internal SSRS management database, _this is **not** the ODS Database!_
+The Report server stores all report server content and application data in a database. This is the internal SSRS management database, _this is **not** the ODS Database!_.
+You may keep on all default settings here if you want.
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/ssrs-and-reports.md:21]: You may keep on all default settings here if you want. - "keep on" = activate?

File: docs/admin/services/reporting-service/reporting-console.md [hunk 13]
```diff
@@ -7,7 +7,7 @@ To set up the **Reporting Console** you will need a machine with a current versi
 ## Installation on a Windows machine
 
 1. Create a directory to host the files of the console application, e.g. _C:\SymbioServices\Reporting\Console_
-2. Copy all files and folders of the application into this directory
+2. After building the reporting project, which contains the Console Service, Copy all the debug/release files and folders of the service into this directory.
```
Comment (JochenLinnemann) [inline docs/admin/services/reporting-service/reporting-console.md:10]: The customer will NOT build the project!

**Review by JochenLinnemann — CHANGES_REQUESTED** [review]

**Comment by JochenLinnemann** [comment]
> I reverted the changes in commits about reporting since they are already in master, should this cause any problems? @JochenLinnemann

I would have expected problems but the system is not showing any deletions so it seems to be OK.

**Comment by JochenLinnemann** [comment]
Please remember the audience of this documentation. It is not "developers at Symbioworld" but customers with no access to the source code. Please move internal FAQ content to our developer Wiki, especially things using the terms "build", "release", "debug".

File: docs/admin/services/reporting-service/faq.md (hunk 1 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/faq.md:79]: Apparently I could not understand the meaning of the original sentence "Can the account Symbio Core is running under do the same"; I mean grammatically it has a problem causing the sentence not being understood, can you please reformulate it for me and I shall change it accordingly in other positions?

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 9 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/reporting-connector.md:12]: where from should the customer copy the files then?

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 10 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/reporting-connector.md:14]: I shall change it

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 11 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/reporting-connector.md:17]: where from should the customer copy the files then?

File: docs/admin/services/reporting-service/ssrs-and-reports.md (hunk 12 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/ssrs-and-reports.md:21]: I just meant he does not need to change other default settings at this step

File: docs/admin/services/reporting-service/faq.md (hunk 1 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/faq.md]: I striked the word "under"

File: docs/admin/services/reporting-service/faq.md (hunk 2 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/faq.md]: done

File: docs/admin/services/reporting-service/faq.md (hunk 3 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/faq.md]: done

File: docs/admin/services/reporting-service/faq.md (hunk 4 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/faq.md]: done

File: docs/admin/services/reporting-service/faq.md (hunk 5 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/faq.md]: done

File: docs/admin/services/reporting-service/reporting-console.md (hunk 13 above)
Comment (KabbaniReem) [reply docs/admin/services/reporting-service/reporting-console.md]: you are right, I deleted all mentions of build/release in this documentation.

**Review by JochenLinnemann — DISMISSED** [review]

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 11 above)
Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/reporting-connector.md]: FTP

File: docs/admin/services/reporting-service/ssrs-and-reports.md (hunk 12 above)
Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/ssrs-and-reports.md:21]: keep on = mach weiter
keep to = bleib bei

File: docs/admin/services/reporting-service/reporting-connector.md (hunk 9 above)
Comment (JochenLinnemann) [reply docs/admin/services/reporting-service/reporting-connector.md]: FTP

**Review by JochenLinnemann — APPROVED** [review]

**Pull request merged**